File: mobile/src/main/java/com/chaos/databinding/models/Repo.java
package com.chaos.databinding.models;


import com.google.gson.annotations.SerializedName;

public class Repo {
    private final String name;
    private final String description;
    private final String language;

    @SerializedName("updated_at")
    private final String updatedAt;

    @SerializedName("stargazers_count")
    private final String stargazersCount;

    @SerializedName("watchers_count")
    private final String watchersCount;

    public Repo(final String name, final String description, final String language,
                final String updatedAt, final String stargazersCount, final String watchersCount) {
        this.name = name;
        this.description = description;
        this.language = language;
        this.updatedAt = updatedAt;
        this.stargazersCount = stargazersCount;
        this.watchersCount = watchersCount;
    }

    public String getName() {
        return this.name;
    }

    public String getDescription() {
        return this.description;
    }

    public String getLanguage() {
        return this.language;
    }

    public String getUpdatedAt() {
        return this.updatedAt;
    }

    public String getStargazersCount() {
        return this.stargazersCount;
    }

    public String getWatchersCount() {
        return this.watchersCount;
    }
}

File: mobile/src/main/kotlin/io/ashdavies/databinding/models/Repo.kt
package io.ashdavies.databinding.models

import com.squareup.moshi.Json
import com.squareup.moshi.JsonClass

@JsonClass(generateAdapter = true)
internal class Repo(
    @Json(name = "name") val name: String,
    @Json(name = "description") val description: String,
    @Json(name = "language") val language: String,
    @Json(name = "updated_at") val updatedAt: String,
    @Json(name = "stargazers_count") val stargazersCount: String,
    @Json(name = "watchers_count") val watchersCount: String
)
